optimizely/event/event_factory.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2019 Optimizely
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .user_event import ConversionEvent, ImpressionEvent
+from .payload import Decision, EventBatch, Snapshot, SnapshotEvent, Visitor, VisitorAttribute
+from .log_event import LogEvent
```

Comment:
Should have mentioned earlier, but import order here is incorrect

It should be external packages (alphabetic), internal packages (alphabetic), files from same package (alphabetic)